### PR TITLE
Update version.md

### PR DESCRIPTION
**Pull request by emmyoop**
## What are you changing in this pull request and why?
The format of the version for versionless in cloud has been updated.  These docs reflect those changes.

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-er-update-versionless-version-docs-dbt-labs.vercel.app/reference/commands/version

### CODE REVIEW

File: website/docs/reference/commands/version.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The `--version` command-line flag returns information about the currently instal
 ## Versioning
 To learn more about release versioning for dbt Core, refer to [How dbt Core uses semantic versioning](/docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning). 
 
-If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.xx.yy` format, where the year is the major version (for example, `2024.04.1234`)
+If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format, where the year, month, and day represent the date the version was built (e.g., `2024.10.28+996c6a8`). The suffix provides additional unique identification for each build."
```
Comment (emmyoop) [inline website/docs/reference/commands/version.md]:
```suggestion
If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format, where the year, month, and day represent the date the version was built (e.g., `2024.10.28+996c6a8`). The suffix provides additional unique identification for each build.
```

**Review by jtcohen6 — APPROVED** [review]
Looks good - thank you! Let's merge when this change is confirmed to be live (at least for customers in MT deployments)

File: website/docs/reference/commands/version.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The `--version` command-line flag returns information about the currently instal
 ## Versioning
 To learn more about release versioning for dbt Core, refer to [How dbt Core uses semantic versioning](/docs/dbt-versions/core#how-dbt-core-uses-semantic-versioning). 
 
-If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.xx.yy` format, where the year is the major version (for example, `2024.04.1234`)
+If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format, where the year, month, and day represent the date the version was built (e.g., `2024.10.28+996c6a8`). The suffix provides additional unique identification for each build.
```
Comment (mirnawong1) [inline website/docs/reference/commands/version.md]:
```suggestion
If using [versionless dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless), then `dbt_version` uses the latest (continuous) release version. This also follows semantic versioning guidelines, using the `YYYY.MM.DD+<suffix>` format. The year, month, and day represent the date the version was built (for example, `2024.10.28+996c6a8`). The suffix provides an additional unique identification for each build.
```

**Review by mirnawong1 — APPROVED** [review]
lgtm w a small tweak, thank you @emmyoop !

**Pull request merged**